AlembicImporter/Assets/API_UnitTests/AlembicAPITest.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Collections;
+using System.IO;
+using System.Linq;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.Formats.Alembic.Exporter;
+using UnityEngine.Formats.Alembic.Sdk;
+using UnityEngine.Formats.Alembic.Timeline;
+using UnityEngine.Formats.Alembic.Util;
+using UnityEngine.TestTools;
+
+namespace UnityEditor.Formats.Alembic.Exporter.UnitTests {
+    public class AlembicAPITest {
+        // Loads a given scene
+        IEnumerator SceneLoader (string sceneToLoad) {
+            SceneManagement.EditorSceneManager.LoadScene (sceneToLoad, UnityEngine.SceneManagement.LoadSceneMode.Single);
+            yield return null;
+        }
+        // Sets up generic export location
+        aeConfig SetUpExport (AlembicRecorder recorderTarget) {
+            recorderTarget.settings.OutputPath = "Assets/API_UnitTests/TestAPI.abc";
+            return recorderTarget.settings.Conf;
+        }
+        // Deletes Test File
+        void TestAndScrub () {
+            Assert.That (Application.dataPath + "/API_UnitTests/TestAPI.abc", Is.Not.Null);
+            File.Delete (Application.dataPath + "/API_UnitTests/TestAPI.abc");
```

Comment:
if you use the test base to generate the filepath then you won't need to destroy it as it will be destroyed automatically